docs/_articles/en/v3.x/custom-rules/author.md

```diff
@@ -16,27 +16,14 @@ Because PMD and ESLint custom rules work differently, Code Analyzer deals with e
 
 ### Write PMD Custom Rules
 
-To be compatible with Code Analyzer, your PMD custom rules must meet these criteria.
+To be compatible with Code Analyzer, your PMD custom rules must meet these guidelines.
 
-* PMD Rules must be XPath-based or Java-based. 
-* Define rules in XML files with a path that matches this format: ```<some base dir>/category/<language>/<filename>.xml```.
-* XPath-based rules can be contained in standalone XML files.
-* Java-based rules must be compiled and bundled into a JAR.
-* Custom rulesets consisting of references to existing rules can be contained in standalone XML files with a path that matches this format: ```<some base dir>/rulesets/<language>/<filename>.xml```.
+* Declare your new XPath or Java-based rules in custom category XML files using this format: `<some base dir>/category/<language>/<filename>.xml`. For more info, read PMD’s [XML rule definition](https://docs.pmd-code.org/latest/pmd_userdocs_extending_writing_rules_intro.html#xml-rule-definition) documentation.
+* Combine your custom rules and PMD's built-in rules into custom rulesets in XML files using this format: `<some base dir>/rulesets/<language>/<filename>.xml`. For more info, read PMD’s [Making rulesets](https://docs.pmd-code.org/latest/pmd_userdocs_making_rulesets.html#referencing-a-single-rule).
+* Register custom rulesets or [XPath-only](https://docs.pmd-code.org/latest/pmd_userdocs_extending_writing_xpath_rules.html) custom categories with Code Analyzer as standalone XML files.
+* Compile your Java-based rules, and bundle them into a JAR along with your rule declaration files, then register that JAR with Code Analyzer. We recommend using a build tool such as [Maven](https://maven.apache.org/plugins/maven-jar-plugin/) or [Gradle](https://docs.gradle.org/current/dsl/org.gradle.api.tasks.bundling.Jar.html) to manage the dependency on PMD and automate the process of building the JAR. Use our <link>example repo</link> as a scaffold to build your own project.
```

Comment:
Replace `<link>example repo</link>` with
`[example repo](https://github.com/forcedotcom/sfdx-scanner/tree/dev/sample-code/pmd-example-rules)`. Once we merge the PR with the sample code, that should link to the sample code's directory, and the README should be displayed automatically by Github.